mappings/net/minecraft/world/biome/BiomeEffects.mapping

```diff
@@ -3,15 +3,24 @@ CLASS net/minecraft/class_4763 net/minecraft/world/biome/BiomeEffects
 	FIELD field_22068 waterColor I
 	FIELD field_22069 waterFogColor I
 	FIELD field_22070 particleConfig Ljava/util/Optional;
+	FIELD field_22491 loopSound Ljava/util/Optional;
+	FIELD field_22492 moodSound Ljava/util/Optional;
+	FIELD field_22493 additionsSound Ljava/util/Optional;
```

Comment:
Additions sound?

Reply:
Yes, I think it is played like in caves? This type of sounds all have ids ending with `_additions` for vanilla nether biomes with such sound effects (same for loop/mood above)

Reply:
wait a sec, will add javadocs specifying in the getter

Reply:
Documented in the latest commit.